product_docs/docs/pem/9/troubleshooting.mdx

```diff
@@ -89,3 +89,16 @@ In some situations, you might need to uninstall the PEM server, reinstall it, an
     /usr/edb/pem/bin/configure-pem-server.sh
     ```
 
+## Editing Apache HTTPD configuration file (PEM client not displaying)
+
+If the PEM client is not displaying, you might see the following HTTPD log in the `/var/log/httpd/error_log` file:
+
+```text
+Truncated or oversized response headers received from daemon process 'edbpem': /usr/edb/pem/web/pem.wsgi
+```
+
+In that case, you must add the following statement at the bottom of the Apache `httpd.conf` file located in `/etc/httpd/conf` folder:
+
+```shell
+WSGIApplicationGroup %{GLOBAL}
+```
```

Comment:
need to add a step to restart the httpd after this change

Reply:
Added